bulletroboy/forces_mapper.py

```diff
@@ -0,0 +1,156 @@
+import rclpy
+from rclpy.node import Node
+
+# from sensor_msgs.msg import JointState
+from roboy_simulation_msgs.msg import Collision
+import yaml
+from roboy_simulation_msgs.srv import LinkIdFromName
+from roboy_simulation_msgs.srv import LinkNameFromId
+from asgiref.sync import async_to_sync
+
+class ForcesMapper(Node):
+    def __init__(self):
+        super().__init__('forces_imitator')
+
+        self.result = None
+
+        self.robotCollisionSubscription = self.create_subscription(
```

Comment:
No need to give the subscription a name, right? You don't use the name variable afterwards.